src/agent/src/rpc.rs

```diff
@@ -120,7 +124,7 @@ fn ttrpc_error(code: ttrpc::Code, err: impl std::fmt::Debug) -> ttrpc::Error {
     get_rpc_status(code, format!("{:?}", err))
 }
 
-fn is_allowed(req: &impl MessageDyn) -> ttrpc::Result<()> {
+fn config_allows(req: &impl MessageDyn) -> ttrpc::Result<()> {
```

Comment:
I vote for replacing the agent config. But, as far as I know, there hasn't been a community decision to remove it yet.

Reply:
I'd remove the agent config at the moment we have the tests in place for this one.

Reply:
Yeah I think it is acceptable to remove at agent config as long as doing so won't cause regressions. At the moment I guess it would given we require systemd.